lib/ansible/utils/module_docs_fragments/aws.py

```diff
@@ -74,7 +81,7 @@ class ModuleDocFragment(object):
     C(AWS_SECRET_ACCESS_KEY) or C(AWS_SECRET_KEY) or C(EC2_SECRET_KEY),
     C(AWS_SECURITY_TOKEN) or C(EC2_SECURITY_TOKEN),
     C(AWS_REGION) or C(EC2_REGION)
-  - Ansible uses the boto configuration file (typically ~/.boto) if no
+  - Ansible uses the boto configuration file (typically ~/.boto or ~/.aws) if no
```

Comment:
`Ansible uses the boto configuration file (typically C(~/.boto) or C(~/.aws)) if no`

Reply:
`C(~/.aws/config) and C(~/.aws/credentials)`, really